ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -329,6 +329,8 @@ let params_of_backend backend =
         (path, [])
     | _, _, {uri} :: _, xendisk :: _ ->
         (uri, [("qemu-params", xendisk.Storage_interface.params)])
+    | _, _, _, xendisk :: _ ->
+        ("", [("qemu-params", xendisk.Storage_interface.params)])
```

Comment:
Why is the URI replaced by the empty string?

Reply:
No support for spawning nbd I assume